index.d.ts

```diff
@@ -555,7 +555,12 @@ declare namespace tracer {
         /**
          * Specifies a regex that will redact sensitive source values in vulnerability reports.
          */
-        redactionValuePattern?: string
+        redactionValuePattern?: string,
+
+        /**
+         * Specifies the verbosity of the send telemetry. Default 'INFORMATION'
+         */
+        telemetryVerbosity?: string
```

Comment:
Not directly related with this changes, but it wasn't added in definitions and test.ts when it was implemented.

Reply:
maybe worth having the iast options in a variable so you don't need to copy the definition twice ?